Data Structures/Linked Lists/C++/Doubly_Linked_List.cpp

```diff
@@ -0,0 +1,93 @@
+#include <iostream>
+#include <conio.h>
```

Comment:
why did you include `conio.h`? it is not POSIX function. so cannot use on Linux or OS X. it is only available on Visual Studio

Reply:
@punkyoon is right. He seems to be using it to make a function call to getch(). You can remove that call to getch() as well as the header file.

Reply:
yes, made these changes